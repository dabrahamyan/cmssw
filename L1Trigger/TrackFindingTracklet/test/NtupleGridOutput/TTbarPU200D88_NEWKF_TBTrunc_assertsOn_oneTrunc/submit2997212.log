000 (2997212.000.000) 09/11 17:08:26 Job submitted from host: <188.185.121.235:9618?addrs=188.185.121.235-9618+[2001-1458-d00-49--100-36e]-9618&alias=bigbird22.cern.ch&noUDP&sock=schedd_159281_1896>
...
040 (2997212.000.000) 09/11 17:09:36 Started transferring input files
	Transferring to host: <128.142.132.71:9618?addrs=128.142.132.71-9618+[2001-1458-301-5a--100-3b]-9618&alias=b7g12p3434.cern.ch&noUDP&sock=slot1_9_3446_c4c5_72794>
...
040 (2997212.000.000) 09/11 17:09:36 Finished transferring input files
...
001 (2997212.000.000) 09/11 17:09:37 Job executing on host: <128.142.132.71:9618?addrs=128.142.132.71-9618+[2001-1458-301-5a--100-3b]-9618&alias=b7g12p3434.cern.ch&noUDP&sock=startd_2950_0879>
...
006 (2997212.000.000) 09/11 17:09:45 Image size of job updated: 45488
	33  -  MemoryUsage of job (MB)
	33792  -  ResidentSetSize of job (KB)
...
006 (2997212.000.000) 09/11 17:14:45 Image size of job updated: 2451864
	1025  -  MemoryUsage of job (MB)
	1048708  -  ResidentSetSize of job (KB)
...
006 (2997212.000.000) 09/11 17:34:47 Image size of job updated: 2624304
	1025  -  MemoryUsage of job (MB)
	1048708  -  ResidentSetSize of job (KB)
...
006 (2997212.000.000) 09/11 17:42:41 Image size of job updated: 6714364
	1025  -  MemoryUsage of job (MB)
	1048708  -  ResidentSetSize of job (KB)
...
040 (2997212.000.000) 09/11 17:42:41 Started transferring output files
...
040 (2997212.000.000) 09/11 17:42:41 Finished transferring output files
...
005 (2997212.000.000) 09/11 17:42:41 Job terminated.
	(1) Normal termination (return value 139)
		Usr 0 00:28:12, Sys 0 00:00:57  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:28:12, Sys 0 00:00:57  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	52265  -  Run Bytes Sent By Job
	694  -  Run Bytes Received By Job
	52265  -  Total Bytes Sent By Job
	694  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :     0.93        1         1 
	   Disk (KB)            :    73           1   1708957 
	   Memory (MB)          :  1025        2000      2000 

	Job terminated of its own accord at 2023-09-11T15:42:41Z.
...
