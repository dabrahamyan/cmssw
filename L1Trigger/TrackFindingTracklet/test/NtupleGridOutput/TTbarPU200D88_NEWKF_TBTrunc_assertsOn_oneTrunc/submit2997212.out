using geometry D88 (tilted)
Mon Sep 11 17:42:00 CEST 2023
Thread 7 (Thread 0x2b38562dd700 (LWP 184) "cmsRun"):
#0  0x00002b38227b9b3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b38227b9bcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b38227b9c6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b3828b19576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002b3828b19639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002b38227b3ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b3822ac6b0d in clone () from /lib64/libc.so.6
Thread 6 (Thread 0x2b38560dc700 (LWP 183) "cmsRun"):
#0  0x00002b38227b9b3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b38227b9bcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b38227b9c6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b3828b19576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002b3828b19639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002b38227b3ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b3822ac6b0d in clone () from /lib64/libc.so.6
Thread 5 (Thread 0x2b3855edb700 (LWP 182) "cmsRun"):
#0  0x00002b38227b9b3b in do_futex_wait.constprop () from /lib64/libpthread.so.0
#1  0x00002b38227b9bcf in __new_sem_wait_slow.constprop.0 () from /lib64/libpthread.so.0
#2  0x00002b38227b9c6b in sem_wait@@GLIBC_2.2.5 () from /lib64/libpthread.so.0
#3  0x00002b3828b19576 in XrdCl::JobManager::RunJobs() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002b3828b19639 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#5  0x00002b38227b3ea5 in start_thread () from /lib64/libpthread.so.0
#6  0x00002b3822ac6b0d in clone () from /lib64/libc.so.6
Thread 4 (Thread 0x2b3855cda700 (LWP 181) "cmsRun"):
#0  0x00002b38227bae9d in nanosleep () from /lib64/libpthread.so.0
#1  0x00002b382885610f in XrdSysTimer::Wait(int) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#2  0x00002b3828a872d3 in XrdCl::TaskManager::RunTasks() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#3  0x00002b3828a87429 in RunRunnerThread () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdCl.so.3
#4  0x00002b38227b3ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b3822ac6b0d in clone () from /lib64/libc.so.6
Thread 3 (Thread 0x2b3855ad9700 (LWP 180) "cmsRun"):
#0  0x00002b3822ac70e3 in epoll_wait () from /lib64/libc.so.6
#1  0x00002b38288505c2 in XrdSys::IOEvents::PollE::Begin(XrdSysSemaphore*, int&, char const**) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#2  0x00002b382884cd4d in XrdSys::IOEvents::BootStrap::Start(void*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#3  0x00002b3828855818 in XrdSysThread_Xeq () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/external/slc7_amd64_gcc10/lib/libXrdUtils.so.3
#4  0x00002b38227b3ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b3822ac6b0d in clone () from /lib64/libc.so.6
Thread 2 (Thread 0x2b38471ee700 (LWP 173) "cmsRun"):
#0  0x00002b38227bb1d9 in waitpid () from /lib64/libpthread.so.0
#1  0x00002b38291abad7 in edm::service::cmssw_stacktrace_fork() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#2  0x00002b38291ac6ca in edm::service::InitRootHandlers::stacktraceHelperThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#3  0x00002b382238ff90 in std::execute_native_thread_routine (__p=0x2b3831867060) at ../../../../../libstdc++-v3/src/c++11/thread.cc:80
#4  0x00002b38227b3ea5 in start_thread () from /lib64/libpthread.so.0
#5  0x00002b3822ac6b0d in clone () from /lib64/libc.so.6
Thread 1 (Thread 0x2b3824b49e40 (LWP 155) "cmsRun"):
#0  0x00002b3822abbddd in poll () from /lib64/libc.so.6
#1  0x00002b38291abdcf in full_read.constprop () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#2  0x00002b38291ac79c in edm::service::InitRootHandlers::stacktraceFromThread() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#3  0x00002b38291af24b in sig_dostack_then_abort () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/pluginFWCoreServicesPlugins.so
#4  <signal handler called>
#5  0x00002b385351d3b5 in std::_Sp_counted_base<(__gnu_cxx::_Lock_policy)2>::_M_release() () from /afs/cern.ch/user/d/dabraham/private/test/tempCMSSWs/three/lib/slc7_amd64_gcc10/pluginTrackFindingTrackletPlugins.so
#6  0x00002b385354e386 in trklet::ProducerKFout::produce(edm::Event&, edm::EventSetup const&) () from /afs/cern.ch/user/d/dabraham/private/test/tempCMSSWs/three/lib/slc7_amd64_gcc10/pluginTrackFindingTrackletPlugins.so
#7  0x00002b38203dc1e3 in edm::stream::EDProducerAdaptorBase::doEvent(edm::EventTransitionInfo const&, edm::ActivityRegistry*, edm::ModuleCallingContext const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#8  0x00002b38203bec3f in edm::WorkerT<edm::stream::EDProducerAdaptorBase>::implDo(edm::EventTransitionInfo const&, edm::ModuleCallingContext const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#9  0x00002b382030eb25 in decltype ({parm#1}()) edm::convertException::wrap<edm::Worker::runModule<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*)::{lambda()#1}>(edm::Worker::runModule<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*)::{lambda()#1}) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#10 0x00002b382030ee13 in std::__exception_ptr::exception_ptr edm::Worker::runModuleAfterAsyncPrefetch<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >(std::__exception_ptr::exception_ptr, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::TransitionInfoType const&, edm::StreamID, edm::ParentContext const&, edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1>::Context const*) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#11 0x00002b3820311546 in edm::Worker::RunModuleTask<edm::OccurrenceTraits<edm::EventPrincipal, (edm::BranchActionType)1> >::execute() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#12 0x00002b38200aa879 in tbb::detail::d1::function_task<edm::WaitingTaskList::announce()::{lambda()#1}>::execute(tbb::detail::d1::execution_data&) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreConcurrency.so
#13 0x00002b3821ba4bec in tbb::detail::r1::task_dispatcher::local_wait_for_all<false, tbb::detail::r1::external_waiter> (waiter=..., t=0x2b382623a500, this=0x2b3826267700) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.h:322
#14 tbb::detail::r1::task_dispatcher::local_wait_for_all<tbb::detail::r1::external_waiter> (waiter=..., t=<optimized out>, this=0x2b3826267700) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.h:463
#15 tbb::detail::r1::task_dispatcher::execute_and_wait (t=<optimized out>, wait_ctx=..., w_ctx=...) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/task_dispatcher.cpp:168
#16 0x00002b382028922d in edm::FinalWaitingTask::wait() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#17 0x00002b3820271310 in edm::EventProcessor::processRuns() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#18 0x00002b382027dda1 in edm::EventProcessor::runToCompletion() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw/CMSSW_12_6_0_pre5/lib/slc7_amd64_gcc10/libFWCoreFramework.so
#19 0x000000000040a266 in tbb::detail::d1::task_arena_function<main::{lambda()#1}::operator()() const::{lambda()#1}, void>::operator()() const ()
#20 0x00002b3821b930eb in tbb::detail::r1::task_arena_impl::execute (ta=..., d=...) at /data/cmsbld/jenkins/workspace/auto-builds/CMSSW_12_6_0_pre3-slc7_amd64_gcc10/build/CMSSW_12_6_0_pre3-build/BUILD/slc7_amd64_gcc10/external/tbb/v2021.5.0-36aff7df349e0716374b1668ccd18e17/tbb-v2021.5.0/src/tbb/arena.cpp:698
#21 0x000000000040b094 in main::{lambda()#1}::operator()() const ()
#22 0x000000000040971c in main ()
